index.html

```diff
@@ -0,0 +1,72 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Portfolio Project</title>
+    <script
+      src="https://kit.fontawesome.com/1572c8e00b.js"
+      crossorigin="anonymous"
+    ></script>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <header class="toolbar">
+      <div class="mylogo">
+        <a href="index.html">Juma</a>
+      </div>
+      <nav class="nav">
+        <ul>
+          <li><a href="#">Portfolio</a></li>
+          <li><a href="#">About Me</a></li>
+          <li><a href="#">Contact</a></li>
+        </ul>
+      </nav>
+      <div class="toggle" id="toggle">
+        <div class="menu"></div>
+      </div>
```

Comment:
[OPTIONAL] Love the animation used here, looks fire 🔥 but the hamburger should be located inside of the `nav` tag for semantic reasons and indexing as well. Look to address that 💯